ddtrace/contrib/starlette/patch.py

```diff
@@ -129,3 +140,24 @@ def traced_handler(wrapped, instance, args, kwargs):
         )
 
     return wrapped(*args, **kwargs)
+
+
+@with_traced_module
+def _trace_background_tasks(module, pin, wrapped, instance, args, kwargs):
+    pin = Pin.get_from(module)
+    if not pin or not pin.enabled():
+        return wrapped(*args, **kwargs)
```

Comment:
Can be removed (already checked in both `FastAPI` and `Starlette`)

Reply:
Good catch 🤝